.github/workflows/accept.yml

```diff
@@ -0,0 +1,173 @@
+name: Fallback Install, Build & Package & Prebuilt Install (Merge)
+
+on: 
+  push: 
+    branches: 
+      # triggered by pull request approval triggers which is a merge to default branch
+      - master
+
+  workflow_dispatch:
+    inputs: 
+      target-test:
+        required: false
+        description: 'Run Tests on Target Group? (type: yes)'
+        default: false
+
+jobs:
+  # entry point 1: fallback group
+  load-fallback-group:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout ${{ github.ref }}
+      uses: actions/checkout@v2
+
+      # *** comments for: fallback-group.json
+      # contains images that are able to build "out of the box"
+    - name: Load target group data
+      id: set-matrix
+      run: .github/scripts/matrix-from-json.sh .github/config/fallback-group.json
+
+  fallback-group-install:
+    runs-on: ubuntu-latest
+    needs: load-fallback-group
+    strategy:
+      matrix: ${{fromJson(needs.load-fallback-group.outputs.matrix)}}
+
+    env:
+      AO_TOKEN_PROD: ${{ secrets.AO_TOKEN_PROD }}
+
+    steps:
+      - name: Checkout ${{ github.ref }}
+        uses: actions/checkout@v2
+
+      - name: Create Container ${{ matrix.image }}
+        id: create
+        run: .github/scripts/container-from-matrix-input.sh ${{ matrix.image }}
+
+      # the current version package has yet to be built.
+      # it will fallback to build from source
+      - name: NPM Install (to fallback)
+        run: docker exec ${{ steps.create.outputs.containerId }} npm install --unsafe-perm
+
+      - name: Run tests
+        run: docker exec ${{ steps.create.outputs.containerId }} npm test
```

Comment:
When test fail, the workflow fails (and will stop other matrix jobs).

Reply:
do you want all of them to stop?
or `fail-fast: false`

Reply:
In such matrix tests I think it would be better not to terminate other runs when one of the runs failed. Reason for this is that, in general, we would want to see if there are any issues with any of the supported configurations and these configurations are practically all independent of each other. If one failing tests terminates the others, it might take some iterations until one can determine which of the configurations actually pass and one would never get a clear matrix like this:
```
config        test result
config_1      pass
config_2      fail
..
config_n      pass
```

Reply:
Agreed. Missed that.
`fail-fast: false` it should be.

Reply:
All strategies are now fail-fast: false. Tested and works as expected. GitHub UI dropdown though, still has only a single option - re-run all jobs